Dispatchfile

```diff
@@ -1,7 +1,64 @@
 #!mesosphere/dispatch-starlark:v0.6
 # vi:syntax=python
 load("github.com/mesosphere/dispatch-catalog/starlark/stable/pipeline@0.0.7", "cron")
+load("github.com/mesosphere/cluster-claim-controller/starlark/claim@master",
+     "make_kind_cluster_claim", "fetch_kubeconfig", "release_claim")
 load("github.com/mesosphere/dispatch-tasks/bump_charts/bump_charts@master", "bump_charts")
+load("github.com/mesosphere/dispatch-catalog/starlark/stable/pipeline@0.0.4", "gitResource", "pullRequest", "push", "volume", "resourceVar", "secretVar")
 
+git = "src-git"
+gitResource(git, url="$(context.git.url)", revision="$(context.git.commit)")
+
+cluster_name = "kubeaddons"
+kind_cluster=make_kind_cluster_claim(cluster_name, deps=[git])
+
+task("integration-test", inputs=[git],
+     deps=[kind_cluster],
+     steps=[
+       fetch_kubeconfig(cluster_name, git, name="fetch-config"),
+       k8s.corev1.Container(
+       name = "integration-tester",
+              workingDir = "/workspace/{}/".format(git),
+              image = "mesosphere/kubeaddons-ci:latest",
+              command = ["/bin/bash", "-c"],
+              args = [
+                  """
+                  set -euo pipefail
+                  export GIT_TERMINAL_PROMPT=1
+                  export GOPRIVATE=github.com/mesosphere/kubeaddons
+                  ssh-keyscan -t rsa github.com > /etc/ssh/known_hosts
+                  git config --global url."https://$GITHUB_TOKEN:@github.com/".insteadOf "https://github.com/"
+                  echo $PWD
+                  ls -a
+                  make kind-test
+                  """
+              ],
+              resources = k8s.corev1.ResourceRequirements(
+              limits = {
+                "cpu": k8s.resource_quantity("8000m"),
```

Comment:
Why 8 cpu and 6 gi of memory?

Reply:
The TC test is the required one. We'll fix the Dispatch related issues in subsequent PRs.